flocker/node/agents/ebs.py

```diff
@@ -58,11 +100,95 @@ def ec2_client(region, zone, access_key_id, secret_access_key):
     config.set('Boto', 'num_retries', BOTO_NUM_RETRIES)
     config.set('Boto', 'metadata_service_num_attempts', BOTO_NUM_RETRIES)
 
+    # Set Boto debug level to BOTO_DEBUG_LEVEL:
+    # "1": basic debug messages
+    config.set('Boto', 'debug', BOTO_DEBUG_LEVEL)
```

Comment:
:question: Are we setting global config options here? Or just for the scope of this connection? It looks global, which seems a bit drastic? 
I guess it doesn't matter too much but if some other library in the agent process was also using boto, it'd be affected by this too, right? Create a followup issue if you agree that this is an issue and if there's a better way.